operation.go

```diff
@@ -629,30 +629,38 @@ func (operation *Operation) ParseRouterComment(commentLine string) error {
 	return nil
 }
 
+type SecurityMap map[string][]string
+
 // ParseSecurityComment parses comment for given `security` comment string.
 func (operation *Operation) ParseSecurityComment(commentLine string) error {
+	var securityMap SecurityMap = SecurityMap{}
 	securitySource := commentLine[strings.Index(commentLine, "@Security")+1:]
-	l := strings.Index(securitySource, "[")
-	r := strings.Index(securitySource, "]")
+	for _, secOption := range strings.Split(securitySource, "||") {
+		secOption = strings.TrimSpace(secOption)
+		operation.ProcessSecurityOption(secOption, securityMap)
+	}
+	operation.Security = append(operation.Security, securityMap)
+	return nil
+}
+
+func (operation *Operation) ProcessSecurityOption(securityOption string, securityMap SecurityMap) {
```

Comment:
We don't need additional export. Even make this a private function even move the code inside the for loop.